homeassistant/components/wink.py

```diff
@@ -66,7 +69,26 @@
 SERVICE_RENAME_DEVICE = 'rename_wink_device'
 SERVICE_DELETE_DEVICE = 'delete_wink_device'
 SERVICE_SET_PAIRING_MODE = 'pair_new_device'
-
+SERVICE_SET_CHIME_VOLUME = "wink_set_chime_volume"
+SERVICE_SET_SIREN_VOLUME = "wink_set_siren_volume"
+SERVICE_ENABLE_CHIME = "wink_enable_chime"
+SERVICE_SET_SIREN_TONE = "wink_set_siren_tone"
+SERVICE_SET_SIREN_AUTO_SHUTOFF = "wink_siren_set_auto_shutoff"
+SERVICE_SET_SIREN_STROBE_ENABLED = "wink_set_siren_strobe_enabled"
+SERVICE_SET_CHIME_STROBE_ENABLED = "wink_set_chime_strobe_enabled"
+SERVICE_ENABLE_SIREN = "wink_enable_siren"
+
+ATTR_VOLUME = "volume"
+ATTR_TONE = "tone"
+ATTR_ENABLED = "enabled"
+ATTR_AUTO_SHUTOFF = "auto_shutoff"
+
+VOLUMES = ["low", "medium", "high"]
+TONES = ["doorbell", "fur_elise", "doorbell_extended", "alert",
```

Comment:
Shouldn't this be stored in pywink ?

Reply:
These are the valid values that can be set in the service. If it was in python-wink the device would have to be created before the valid states would be known. I don't know I could see putting it there... Having it hear allows the service to error sooner and the user to know in the frontend it isn't valid though?